Frameworks/CoreFoundation/Locale.subproj/CFNumberFormatter.c

```diff
@@ -786,7 +786,7 @@ void CFNumberFormatterSetProperty(CFNumberFormatterRef formatter, CFStringRef ke
 #if U_ICU_VERSION_MAJOR_NUM >= 55
         __CFGenericValidateType(value, CFNumberGetTypeID());
         CFNumberGetValue((CFNumberRef)value, kCFNumberSInt32Type, &n);
-        __cficu_unum_setContext(formatter->_nf, n, &status);
+        __cficu_unum_setContext(formatter->_nf, (UDisplayContext)n, &status);
```

Comment:
// WINOBJC: WinObjC seems to have stricter type-checking than the reference platform,
        // so an explicit cast is required here

---
In reply to: [71390930](https://github.com/Microsoft/WinObjC/pull/668#discussion_r71390930) [](ancestors = 71390930,71390074)